pkg/models/user.go

```diff
@@ -79,13 +79,13 @@ var adminUID = uuid.Must(uuid.NewRandom()).String()
 
 //DefaultUser is the admin user created by default
 var DefaultUser = &UserCredentials{
-	UID:      &adminUID,
-	Name:     &types.DefaultName,
-	Email:    &types.DefaultEmail,
-	UserName: &types.DefaultUserName,
-	Password: &types.DefaultUserPassword,
-	Role:     RoleAdmin,
-	Kind:     LocalAuth,
+	UID:             &adminUID,
+	Name:            &types.DefaultName,
+	UnverifiedEmail: &types.DefaultUserName,
```

Comment:
If we keep this empty, will it cause any issue?

Reply:
It will not cause any issue in the backend but it can cause some issue in UI as we will have to treat this case differently while signing up because rest all the users that will signup will surely contain this key

Reply:
But, admin does not "sign up", right?

Reply:
Oh If the admin need not to signup then this is fine. Then we can remove this change. Normally we were making admin signup like a regular user.

Reply:
Aren't we creating the admin user during initialisation? Admin can only signin, not signup.
Is that not correct?

Reply:
Yes but what we are doing previously is that we just create the admin credentials and assign his role as `admin` afterwards admin has to go through the same steps as a user i.e. verify his email and enter company details

Reply:
Admin need not verify or give company/role details as a mandatory case. I think we should mark the onboarding status as UnverifiedComplete during creation time (or in DB for already-created cases).